server/src/test/java/org/elasticsearch/common/logging/JsonThrowablePatternConverterTests.java

```diff
@@ -37,18 +36,26 @@ public void testNoStacktrace() throws IOException {
     }
 
     public void testStacktraceWithJson() throws IOException {
-        String json = """
-            {
-              "terms": {
-                "user": [
-                  "u1",
-                  "u2",
-                  "u3"
-                ],
-                "boost": 1.0
-              }
-            }\
-            """;
+
+        String json = "{"
+            + LINE_SEPARATOR
+            + "  \"terms\" : {"
+            + LINE_SEPARATOR
+            + "    \"user\" : ["
+            + LINE_SEPARATOR
+            + "      \"u1\","
+            + LINE_SEPARATOR
+            + "      \"u2\","
+            + LINE_SEPARATOR
+            + "      \"u3\""
+            + LINE_SEPARATOR
+            + "    ],"
+            + LINE_SEPARATOR
+            + "    \"boost\" : 1.0"
+            + LINE_SEPARATOR
+            + "  }"
+            + LINE_SEPARATOR
+            + "}";
```

Comment:
Would this be any clearer?

```suggestion
        String json = """
            {
              "terms": {
                "user": [
                  "u1",
                  "u2",
                  "u3"
                ],
                "boost": 1.0
              }
            }\
            """.lines().collect(Collectors.joining(System.lineSeparator()));
```